common/src/model_registration/register.py

```diff
@@ -80,6 +82,9 @@ def parse_args():
         default=None,
     )
     args = parser.parse_args()
+    tc.track_event(name="FM_import_pipeline_debug_logs", 
```

Comment:
We might want to have all of these being logged in a separate function with try/except so that failures are handled gracefully like @elesel is suggesting.

Reply:
Done